### PR TITLE
some fedora lovers opinions & when it is not p…

### LINKED ISSUE #6: [Fedora] Intended Users
- Real life use cases.
  - Cases where it might not be beneficial to use this specific distribution.

### PR DESCRIPTION
**Pull request by KenzyAdel**
closes #6

### CODE REVIEW

File: Fedora/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##Real Life use case
+---
+[Tom Kunz](https://www.trustradius.com/reviews/fedora-linux-2021-06-29-11-16-36) ~ a Solid Rock Technologies Engineer: 
+**"Fedora is excellent for test-lab/R&D deployment"**
+We have had Fedora Linux in testing for a while, and done some limited deployments. It's not our first-choice system, but it is definitely something we have used for certain applications and systems.
+You can see more use cases [here](https://www.reddit.com/r/linuxmasterrace/comments/gx5i3b/what_are_the_real_use_cases_for_fedora/).
+
+##When it might not be beneficial to use fedora?
+---
+If you don't like changes and want a very stable distribution then fedora will not be the best choice for you.
```
Comment (thisisamna) [inline Fedora/README.md]: Capital F

File: Fedora/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+##Real Life use case
+---
+[Tom Kunz](https://www.trustradius.com/reviews/fedora-linux-2021-06-29-11-16-36) ~ a Solid Rock Technologies Engineer: 
+**"Fedora is excellent for test-lab/R&D deployment"**
+We have had Fedora Linux in testing for a while, and done some limited deployments. It's not our first-choice system, but it is definitely something we have used for certain applications and systems.
+You can see more use cases [here](https://www.reddit.com/r/linuxmasterrace/comments/gx5i3b/what_are_the_real_use_cases_for_fedora/).
+
+##When it might not be beneficial to use fedora?
```
Comment (thisisamna) [inline Fedora/README.md]: Capital F and space

File: Fedora/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+##Real Life use case
```
Comment (thisisamna) [inline Fedora/README.md]: Space

**Review by thisisamna — CHANGES_REQUESTED** [review]
Great work again! Just add the spaces in the titles, and make sure to capitalize Fedora.

**Comment by KenzyAdel** [comment]
All is done here. Please, view changes ASAP and tell me if there is any notes

**Review by thisisamna — APPROVED** [review]
LGTM!

**Pull request merged**